numba/cuda/cudamath.py

```diff
@@ -127,5 +127,5 @@ class Math_isnan(ConcreteTemplate):
 class Math_modf(ConcreteTemplate):
     cases = [
         signature(types.UniTuple(types.float64, 2), types.float64),
-        signature(types.UniTuple(types.float32, 2), types.float32)
+        signature(types.UniTuple(types.float32, 2), types.float32),
```

Comment:
A stray change seems to have crept in here:
```suggestion
        signature(types.UniTuple(types.float32, 2), types.float32)
```